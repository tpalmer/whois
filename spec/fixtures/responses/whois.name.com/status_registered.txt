
__   _                             ____
| \ | | __ _ _ __ ___   ___       / ___|___  _ __ ___
|  \| |/ _` | '_ ` _ \ / _ \     | |   / _ \| '_ ` _ \
| |\  | (_| | | | | | |  __/  _  | |__| (_) | | | | | |
|_| \_|\__,_|_| |_| |_|\___| (_)  \____\___/|_| |_| |_|
      On a first name basis with the rest of the world.


Get your <a href="http://www.name.com">domains</a> at Name.com.


Domain Name:     name.com
Registrar:       Name.com LLC

Expiration Date: 2019-11-04 00:00:00
Creation Date:   1995-01-03 05:00:00

Name Servers:
	ns1.name.com
	ns3.name.com
	dns01.name.com
	dns02.name.com

REGISTRANT CONTACT INFO
Name.com LLC
DNS Admin
2500 East Second Avenue
Second Floor
Denver
CO
80206
US
Phone:         +1.7202492374
Email Address: dns@name.com

ADMINISTRATIVE CONTACT INFO
Name.com LLC
DNS Admin
2500 East Second Avenue
Second Floor
Denver
CO
80206
US
Phone:         +1.7202492374
Email Address: dns@name.com

TECHNICAL CONTACT INFO
Name.com LLC
DNS Admin
2500 East Second Avenue
Second Floor
Denver
CO
80206
US
Phone:         +1.7202492374
Email Address: dns@name.com

BILLING CONTACT INFO
Name.com LLC
DNS Admin
2500 East Second Avenue
Second Floor
Denver
CO
80206
US
Phone:         +1.7202492374
Email Address: dns@name.com

Timestamp: 1341514945.6921

The Data in the Name.com LLC WHOIS database is provided by Name.com LLC for information purposes, and to assist persons in obtaining information about or related to a domain name registration record.  Name.com LLC does not guarantee its accuracy.  By submitting a WHOIS query, you agree that you will use this Data only for lawful purposes and that, under no circumstances will you use this Data to:  (1) allow, enable, or otherwise support the transmission of mass unsolicited, commercial advertising or solicitations via e-mail (spam); or (2) enable high volume, automated, electronic processes that apply to Name.com LLC (or its systems). Name.com LLC reserves the right to modify these terms at any time.  By submitting this query, you agree to abide by this policy.

Cached on: 2012-07-05T13:02:25-06:00
