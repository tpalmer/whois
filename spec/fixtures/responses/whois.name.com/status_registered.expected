#status
  should: %s == :registered

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("1995-01-03 05:00:00")

#updated_on
  should: %s == nil

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2019-11-04 00:00:00")


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == nil
  should: %s.name         == "Name.com"
  should: %s.organization == "Name.com LLC"
  should: %s.url          == "http://www.name.com"

#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_REGISTRANT
  should: %s[0].id           == nil
  should: %s[0].name         == "DNS Admin"
  should: %s[0].organization == "Name.com LLC"
  should: %s[0].address      == "2500 East Second Avenue\nSecond Floor"
  should: %s[0].city         == "Denver"
  should: %s[0].state        == "CO"
  should: %s[0].zip          == "80206"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "+1.7202492374"
  should: %s[0].fax          == nil
  should: %s[0].email        == "dns@name.com"

#admin_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_ADMIN
  should: %s[0].id           == nil
  should: %s[0].name         == "DNS Admin"
  should: %s[0].organization == "Name.com LLC"
  should: %s[0].address      == "2500 East Second Avenue\nSecond Floor"
  should: %s[0].city         == "Denver"
  should: %s[0].state        == "CO"
  should: %s[0].zip          == "80206"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "+1.7202492374"
  should: %s[0].fax          == nil
  should: %s[0].email        == "dns@name.com"

#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_TECHNICAL
  should: %s[0].id           == nil
  should: %s[0].name         == "DNS Admin"
  should: %s[0].organization == "Name.com LLC"
  should: %s[0].address      == "2500 East Second Avenue\nSecond Floor"
  should: %s[0].city         == "Denver"
  should: %s[0].state        == "CO"
  should: %s[0].zip          == "80206"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "+1.7202492374"
  should: %s[0].fax          == nil
  should: %s[0].email        == "dns@name.com"


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(4)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns1.name.com"
  should: %s[0].ipv4 == nil
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns3.name.com"
  should: %s[1].ipv4 == nil
  should: %s[2] CLASS(nameserver)
  should: %s[2].name == "dns01.name.com"
  should: %s[2].ipv4 == nil
  should: %s[3] CLASS(nameserver)
  should: %s[3].name == "dns02.name.com"
  should: %s[3].ipv4 == nil

